aod-discord-bot.js

```diff
@@ -319,6 +322,20 @@ function messageReply(message, msg, edit) {
 }
 global.messageReply = messageReply;
 
+function sendGlobalNotification(interaction, msg) {
+	if (interaction) {
+		if (!globalNotificationChannel || globalNotificationChannel.name !== config.globalNotificationChannel) {
+			globalNotificationChannel = interaction.guild.channels.cache.find(c => { return c.name === config.globalNotificationChannel; });
+		}
+		if (typeof(msg) === 'object')
+			return globalNotificationChannel.send(interaction, { embeds: [msg] }).catch(() => {});
```

Comment:
I don't think you want to provide interaction here.

Reply:
Resolved in d507beb